var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -90,9 +90,13 @@ def configure_args(self):
             '-des',
             '-Dprefix={0}'.format(prefix),
             '-Dlocincpth=' + self.spec['gdbm'].prefix.include,
-            '-Dloclibpth=' + self.spec['gdbm'].prefix.lib,
+            '-Dloclibpth=' + self.spec['gdbm'].prefix.lib
         ]
 
+        # Prepend default perl @INC path to allow package activation:
+        config_args.append('-Accflags=-DAPPLLIB_EXP=\\"' + join_path(
+                           self.spec.prefix, 'lib', 'perl5') + '\\"')
```

Comment:
I found the answer to my own question at http://spack.readthedocs.io/en/latest/packaging_guide.html#prefix-objects . There are separate `prefix.lib` and `prefix.lib64` attributes. It should be safe to assume they will never change, but we can expect trouble if they do.